src/components/players/list/index.js

```diff
@@ -12,51 +14,83 @@ class List extends React.Component {
     constructor(props) {
         super(props);
 
-        this.ref = createRef();
-
         this.state = {
-            selectedId: null,
-            selectedTeam: this.props.selectedTeam,
+            selectedPlayerId: null,
         };
+
+        this.ref = createRef();
     };
 
     componentDidUpdate(prevProps) {
-        if (prevProps.selectedTeam !== this.props.selectedTeam) {
-            this.setState({ selectedId: null });
-            this.ref.current.scrollTo(0, 0);
+        // console.log(this.props.selectedTeam.isLoading);
+
+        // if (this.props.selectedTeam.isLoading) {
+        //     // this.ref.current.scrollTo(0,0);
+        //     this.ref.current.style.overflow = 'hidden';
+        // } else {
+        //     this.ref.current.style.overflow = 'auto';
+        // }
```

Comment:
I see what ur trying to do here. Much simpler solution to make scroll to top:
```
        if (this.props.selectedTeam.isLoading) {
            this.ref.current.scrollTo(0,0);
        }
````